sycl/doc/extensions/OrderedQueue/OrderedQueue_v2.adoc

```diff
@@ -0,0 +1,31 @@
+= SYCL Proposals: Queue Order Properties
+James Brodman <james.brodman@intel.com>
+v0.2
+:source-highlighter: pygments
+:icons: font
+== Introduction
+This document presents an addition proposed for a future version of the SYCL Specification.  The goal of this proposal is to reduce the complexity and verbosity of using SYCL for programmers.
+
+== Queue Order Properties
+Queues in SYCL are out-of-order by default.  SYCL constructs directed acyclic graphs to ensure tasks are properly ordered based on their data dependences.  However, many programs only require linear DAGs.  The overheads of constructing and managing DAGs are unnecessary for this class of program.  The `in_order` queue property is proposed to serve this class of programs by providing programmer-specified in-order semantics.  This property is used with the existing `property_list` mechanism that is part of the normal SYCL `queue`.  
```

Comment:
Should "task" be "kernel" here?

Reply:
No - because the operation could be a copy, etc.

Reply:
Ah, I see.  Is there any other word in the specification that describes the collection of things that can appear in command group scope?  If not, "task" is fine.

Reply:
I found that the specification describes the behavior of `wait` as "Performs a blocking wait for the completion of all enqueued tasks in the queue.", so what you have here isn't inconsistent with the specification.

I'd still prefer that there was one word used consistently everywhere to capture this concept, but the issue is larger than this PR and we should revisit it later.